config/cht/test/forms/pregnancy.spec.js

```diff
@@ -0,0 +1,136 @@
+const { expect } = require('chai');
+const TestRunner = require('medic-conf-test-harness');
+const path = require('path');
+const harness = new TestRunner({
+  xformFolderPath: path.join(__dirname, '../../forms/app'),
+});
+
+describe('Getting started tests', () => {
+  before(async () => { return await harness.start(); });
+  after(async () => { return await harness.stop(); });
+  beforeEach(async () => { return await harness.clear(); });
+  afterEach(() => { expect(harness.consoleErrors).to.be.empty; });
+
+  it('pregnancy form can be loaded', async () => {
+    await harness.loadForm('pregnancy');
+    expect(harness.state.pageContent).to.include('id="pregnancy"');
+  });
+
+   it('unit test confirming pregnancy with pregnancy followup date', async () => {
+    await harness.setNow('2000-01-01 05:45');
+    // Load the pregnancy form and fill in
+    const result = await harness.fillForm('pregnancy', ['method_lmp'], ['1999-08-01'], [], ['0'], ['yes', '2000-01-15'], ['no', 'no'], ['none', 'no'], ['yes', 'no', 'no', 'no', 'no', 'no', 'no', 'no', 'no', 'no', 'no'], ['no'], ['no'], ['no'], [], ['no'], []);
```

Comment:
Do these form inputs not work with within the pattern of having `form-inputs.js`? Do you capture them within the test because they contain dates?

Reply:
I wrote these tests in the beginning and forgot to move the inputs to `form-inputs.js` later. Will do.

Reply:
Done.